src/test/java/nl/topicus/mssql2monetdb/test/CopyTableTest.java

```diff
@@ -0,0 +1,48 @@
+package nl.topicus.mssql2monetdb.test;
+
+import static org.junit.Assert.*;
+import nl.topicus.mssql2monetdb.CopyTable;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CopyTableTest {
+
+	@Test
+	public void testGenerateCountQuery() {
+		CopyTable table = new CopyTable();
+		
+		// test standard count query based on table from name
+		table.setFromName("test");
+		Assert.assertEquals(table.generateCountQuery(), "SELECT COUNT(*) FROM [test]");
+		
+		table.setFromName(null);
+		
+		// test a custom count query
+		String customCountQuery = "SELECT COUNT(*) FROM myQuery WHERE bla = 'something'";
+		table.setFromCountQuery(customCountQuery);		
+		Assert.assertEquals(table.generateCountQuery(), customCountQuery);	
```

Comment:
Een beetje flauw, maar dit is niet hoe je wilt testen, het verwachtte resultaat moet je altijd 'hardcoden'. Je geeft nu de input direct op als verwacht resultaat.